### PR TITLE
[Live-Site] Created panel component

### LINKED ISSUE #462: [Live-Site] Create the panel component
image
> component to create

Figma Link - https://www.figma.com/file/2VtPGJH61Qp0iarkKW6KTO/Live-Site-Detailed-Designs?node-id=0-1&t=mIdw0NLHXmNrU7Vz-0

### PR DESCRIPTION
**Pull request by satyam73**
### Issue

Closes #462 
### What is the change?

I have added the panel component for live site.

### Is Development Tested?

- [x] Yes
- [ ] No

### Before / After Change Screenshots
image
>Desktop Full Image

image
> Individual Component Image

image
>Mobile Image

**Note :**
What to delete and what not to

- Keep the headings and answer the questions.
- Delete the examples
- Delete the description below headings `What is the change?` & `Before/After Screenshots`

### CODE REVIEW

File: app/styles/icon-button.module.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    background-color: var(--color-lightgrey);
+    cursor: pointer;
+    display: inline-block;
+}
+
+.icon-button--md {
+    border-radius: 50%;
+    padding: 0.51rem;
+    height: 2.81rem;
+    width: 2.81rem;
```
[the remark this answers is not in the document]
Comment (bhtibrewal) [reply app/styles/icon-button.module.css:12]: Its easy to maintain consistency with whole numbers like 2,4 etc you dont have to remeber much, but with this kinda numbers you have to remember them or go and check what did you wrote. Also you have not used variable for it so if change a little at one place you will have to go and manually change at all other places.
With this number multiples are also difficult like 0.51, 1.02, 2.04 these kinda numbers

[the remark this answers is not in the document]
Comment (satyam73) [reply app/styles/icon-button.module.css:12]: Why we will use it in multiples of these numbers any usecases I am missing here?

Also we can create `--sm` , `--lg`, `--xl`, `xxl` if we want any one of these in future.

**Review by rohan09-raj — COMMENTED** [review]
@satyam73 Adding few comments regarding the UI
- Why making these right aligned, why not center aligned. I think it will look much better if it's center aligned
image
- Can we please use same family of icons here like having same stroke width ? Use fortawesome/fontawesome which is already included in the codebase (package.json) https://fontawesome.com/icons/screen-users?f=classic&s=solid
- Have some padding in icons so that icon and icon background look visually good
- In mobile/tablet view, use small font and icons sizes, take reference from the navbar elements and place the values relatively.

**Comment by satyam73** [comment]
> @satyam73 Adding few comments regarding the UI
> 
> * Why making these right aligned, why not center aligned. I think it will look much better if it's center aligned
>   image
> * Can we please use same family of icons here like having same stroke width ? Use fortawesome/fontawesome which is already included in the codebase (package.json) https://fontawesome.com/icons/screen-users?f=classic&s=solid
> * Have some padding in icons so that icon and icon background look visually good
> * In mobile/tablet view, use small font and icons sizes, take reference from the navbar elements and place the values relatively.

The rest things I can do but I wasn't able to find the relatable icons in fontawesome that's why used this.

**Comment by satyam73** [comment]
Hey @rohan09-raj here's the updated screenshots of the same.

image
>Updated Image (Desktop)
image
> Updated Image (Individual Component)
image
> Updated Image (Mobile )

File: app/components/reusables/icon-button.hbs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<button
+  class='icon-button {{@class}}'
+  type='button'
+  {{on 'click' (fn @onClick @id)}}
+  data-test-icon-button={{@id}}
+>
+  <IconifyIcon data-test-icon={{@id}} @icon={{@icon}} />
```
Comment (shubhamsigdar1) [inline app/components/reusables/icon-button.hbs:7]: Is I am right??
This is component and you not created this component function ??

**Review by shubhamsigdar1 — COMMENTED** [review]
great work satyam...can you please answer one of my comment

**Review by bhtibrewal — APPROVED** [review]

File: app/components/reusables/icon-button.hbs (hunk 2 above)
Comment (satyam73) [reply app/components/reusables/icon-button.hbs:7]: Yeah this is component and functions are passed into it as a props

**Review by rohan09-raj — COMMENTED** [review]
Requesting following changes:
- We can adjust the height of live panel relative to navbar in both mobile and web view
- Height of circle buttons should be same as of the height of start button
- Format the CSS with Prettier

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**